src/controllers/profile.js

```diff
@@ -0,0 +1,122 @@
+const User = require('../database/models').User;
+// const model = require('../database/models');
```

Comment:
Consider removing unnecessary commented lines

Reply:
Yeah thank you! I'm looking forward to responding to your feedback soon.